src/main/java/tc/oc/pgm/crafting/CraftingMatchModule.java

```diff
@@ -34,16 +35,16 @@ public void enable() {
     }
 
     for (Recipe recipe : customRecipes) {
-      getMatch().getServer().addRecipe(recipe);
+      PGM.get().getServer().addRecipe(recipe);
     }
   }
 
   @Override
   public void disable() {
-    // Recipe changes affect all worlds on the server, so we make changes at match start/end
+    // Recipe changes affect all worlds on the server, so we make changes at match start/finish
```

Comment:
SportPaper has per world recipe changes, the server recipe methods loop through all of the worlds and set the recipes for each world.